physics/Magnetized Cold Plasma Properties.ipynb

```diff
@@ -0,0 +1,272 @@
s3yuGpi5xv6Xd2/gBGz6WazGTc3N0teExISSE9Pt7pAL2BveZYZ9TLydvZmbOhYZveZjYMp7+eeS2mXmPrLVPov68+Tm5/kj6g/5KdtIYQQwmDHjx/ntddeIygoiPbt2/Pzzz/z3HPPce7cOT744IMizw0Jydv9rWANc4FTp06RmppKixYtyi1uUbzg4GDOnDnDhg0bCAgI4KmnnqJWrVo88MADLF++nMzMTKvGsbc8S6FeDrJys+jbqC9mZWbz2c38Y/0/GPH9CJZFLiMjp3RXmwshhBCi5JKTk/noo4/o3r07TZs25fPPP2fIkCHs37+fPXv2MGnSJKu22wsICKBVq1bMmzfvmuMRERE4OjrSt2/f8noLwkomk4kePXowe/ZsoqOj+fLLL0lOTmb48OH4+/szfvx4duzYUeQY9pZnWfpSDvzd/ZneaTpP3/40C48uZMHRBRyNP8rL219m5u6ZrBqyCk8n+/+1oBBCCGFv1q1bh7+/P2lpabi6ugLg7e1Nr169btp/586dPP/88wwcOJA333yT7t27l3g5RIE33niDAQMG8OijjzJ69Gj27NnDtGnTePLJJ2UPdRsryPPVvL296dChg1Xnu7m5ERYWxqBBg0hJSWH+/PnMmTOHyMhItmzZUuS59pRnKdTLka+rLxNaT+CRlo+w9uRa5h2eRw3XGpYiXWtN5OVIgnyCDI5UCCGEqBwmTpx4w7GQkBAOHDhw0/5t27YlOjqanJycMq+d79evH4sXL+a1115j9uzZ1KpVixdeeIGpU6eWaVxxo8Ly/Ouvv5Z4LH9/fyZNmsSkSZOIiooqtr895VkK9QrgZHbivib3MShwEClZKZbje2L28NC6h2jj14aw4DB6BPSwrHMXQgghxBVjx45l7NixlufWXrTq5eVl6W8LQ4cOZejQoTYZS9xo7NixDBs2rNDcljWP1s6I20uepSqsQEopPJw8LM/PJ5/H09GTPTF72BOzB393f0Y3H82wpsPwdq74K6aFEEIIIYT9kItJDTQwcCCbRmzihfYv0MCrAVEpUfy/Xf+PXot7MXPXTKPDE0IIIYQQBpJC3WBujm6Mbj6alYNX8nGPj7mr9l2kZaehubKdY47OIVfnGhilEEIIIYSoaLL0xU6YlIku9brQpV4Xjscfp5pLNUvbtuRtzFw+k/DgcAYGDsTN0c3ASIUQQgghREWQGXU71MSnCb6uvpbnf6b+yanEU0zbMY2ei3vy3s73uJB8wcAIhRBCCCFEeZNCvRKY4DeBt+95m1Y1W5GUmcTXB7+m79K+TNo6iUOXDhkdnhBCCCGEKAdSqFcCZmWmT8M+RPSLYH6/+fRr1A8TJjae3sjJhJNGhyeEEEIIIcqBrFGvZFrWbMmMmjN4pt0zLItcRu8GvS1tH+75EAeTAyODRlLDtYaBUQohhBBCiLKSQr2S8nPz49FWj1qeJ2QkMOfgHNJz0vl83+f0a9SP8BbhNK/e3MAohRBCCCFEacnSlyrCy8mLj3p8RNf6XcnOzWbFiRWM+H4E49aN44czP5CTm2N0iEIIIYQQogSkUK8ilFK0r92eD7t/yKohqwgPDsfd0Z2d0Tt5astTspZdCCFEpTZ79myUUpZHjRo1CAwM5IUXXiA9Pb3C4zl48CC9e/fGw8ODGjVqMG7cOOLi4io8jqpm9uzZeHl5WfLs5OR0S+dZlr5UQQFeATx353P8q/W/WH58Ocfij9HEp4mlffaB2XQL6GZghEIIIUTpLFq0iHr16hEdHc369et58803SUpK4sMPP6ywGC5cuEDXrl1p3rw5ixcv5vLly0yePJkBAwbwyy+/YDLJPGhZFeQ5KSmJZcuWWfL8xhtvVFgM9pBnKdSrMA8nD8JbhF9zbN/Ffby7613e2/UeIa4hOF9wpkPtDiilDIpSCCGEsF7r1q1p0qQJSUlJ3HfffURGRvLll1/y/vvvF1s4RUZG0rRp0zLH8Pbbb5OVlcX3339PtWp5NyisU6cO99xzD8uXL2fo0KFlfo1bXUGeAXr16mXJ87Rp02wyvjXfC/aQZ/mR7xZTzbkaQ5oMwdHkyIG0A4zfOJ6hK4ey+Nhi0rMr/ldKQgghRFm0bduWtLQ0YmNji+0bFBTEXXfdxaxZs7h06VKpX3PlypX079/fUrwBdOnShYCAAFasWFHqcUXhCvJclrxdunSJWbNmcddddxEUFFRsf3vIsxTqt5gArwBev/t1No7YSH/v/tR0rcnxy8d57dfXGLJiiFx0KoQQolI5deoU3t7e1KhR/LbE69evp2nTpjz33HPUrl2bwYMHs3TpUjIyMqx+vbS0NE6ePEloaOgNbSEhIRw6JDciLA8Fea5evXqJzsvIyGDp0qUMHjyY2rVrM2XKFIKCgli/fn2R59lLnmXpyy2qukt1+lTrw386/4f1p9cz79A82tRqg9lkBiAtO43I+Ehuq3mbwZEKIYSwmVe9S3yKZ3n0fzWhxHEUyMnJITs7m/j4eBYtWsSSJUuYOXMmZrO52HN79+5N7969SU1NZcWKFURERDBq1Cg8PT0ZOXIkDzzwAHfffXeRY8THx6O1xsfH54a26tWrc/To0VK/N1vyfLde4Y0DZkK7cXlf7/waVj114/kFX1ydq8+6wN9/3nRI55ZjYNgneU8u7IE6bUoe9FUK8lywRr0keQbYtm0bc+fOZcGCBaSkpNC7d2/mzJnDfffdh6ura7Hn20uepVC/xTmaHRnQeAD9G/UnOzfbcnz1X6t57dfXuM33NsJbhNOzQU8cTY4GRiqEEEJA8+bX3h9kwoQJPPHEEyUaw83NjdGjRzN69GhiY2P57rvviIiI4LPPPqNx48acOHGi0HO11gA3vbaroE2UXWF5TkpKKvbcwMBA/vrrL+666y5efPFFHnroIXx9fUv0+vaSZynUBZD3jehovlKIZ+Vm4eXkxb7Yffz7p3/j5+bH6OajGd50ONVcqhUxkhBCCLtVipnspKQkPD2tn1cvaf+SWrZsG
```

Comment:
```suggestion
    "plt.xlim(1e6, 200e9)\n",
    "plt.grid(True, which='both')\n",
    "plt.xlim(1e6, 200e9)\n",
```
one less line ;)

Reply:
gah, github is really being a pain here

Reply:
Indeed dealing with notebook in github with plain text is not easy. An inconvenient of notebooks is that they rapidly increase the repository size. On [scikit-rf](https://github.com/scikit-rf), the good practice is to pull "empty" notebooks, for this reason of size. Then the html documentation is built with sphinx from the notebooks.